pages/api/websites/[id]/index.js

```diff
@@ -43,7 +44,7 @@ export default async (req, res) => {
           name,
           domain,
           shareId: shareId ? shareId : newShareId,
-          userId: account ? account.id : +owner || undefined,
+          userId: +owner,
```

Comment:
if owner is not passed in, the + will create a NaN. Need to guard against it.

Reply:
I checked both API post, whether user is admin or non-admin, owner is always passed in.

Can add some validation incase it's manually called.